VocaDbWeb/Content/Styles/ExtLinks.css

```diff
@@ -295,3 +295,15 @@ a.extLink[href^="http://vocaloid.wikia.com/wiki/"],
 a.extLink[href^="https://vocaloid.fandom.com/wiki/"] {
   background: url("/Content/ExtIcons/vocawiki.png") no-repeat left 50%;
 }
+a.extLink[href*=".moegirl.org/"] {
```

Comment:
Please understand that these wildcard matchers have worse performance than prefix matchers. Are all of these required to be wildcards?

Reply:
> Please understand that these wildcard matchers have worse performance than prefix matchers. Are all of these required to be wildcards? If it's just http/https, then better have it has prefix matcher.

I see. Modified some of them.

Reply:
Well that, and it's also safer to make it as specific as possible.